contrib/linux/actions/wait_for_ssh.py

```diff
@@ -17,9 +17,9 @@ def run(self, keyfile, username, hostname, ssh_timeout, retries):
             try:
                 client.connect(hostname=hostname, username=username, pkey=key)
                 return True
-            except Exception, e: 
+            except Exception, e:
                 self.logger.info(e)
                 time.sleep(ssh_timeout)
             time.sleep(20)
```

Comment:
No clue, it looks like some old code leftover which shouldn't be there.

I'll remove it.